apps/examples/st_things/Kconfig

```diff
@@ -52,6 +52,13 @@ config RESET_BUTTON_PIN_NUMBER
 		The default is the reset button pin number 44 for the ARTIK 053.
 		If you use another board, you can set the pin number for the board.
 
+config CONFIG_HANDLE_GPIO_EVENT_NUMBER
+	int "GPIO Number of handle push service"
+	default 42
+	depends on RESET_BUTTON
```

Comment:
Why does this have a dependancy with RESET_BUTTON?

Reply:
I think it will be depend on EXAMPLES_ST_THINGS_LIGHT

Reply:
I think, it should depend on EXAMPLES_ST_THINGS_LIGHT